Update guestbook chart in repo

hickeyma opened the pull request:
charts/guestbook was updated in a previous PR. This updates the chart to 0.2.0 in the Helm101 repo.

hickeyma committed:
Update guestbook chart in repo

charts/guestbook was updated in a previous PR. This updates the chart
in the Helm101 repo.

spzala commented on presentation/demosteps.md:
````diff
@@ -26,7 +26,7 @@ $ helm search helm101
 Deploy: 
 ```bash
-$ helm install helm101/guestbook --name myguestbook --set serviceType=NodePort
+$ helm install helm101/guestbook --name myguestbook --set service.type=NodePort
````
Can we keep serviceType=NodePort as a comment that for chart version 0.1.0 we can use it that way?

hickeyma replied:
Sure. I added a note.

spzala commented on repo/stable/index.yaml:
```diff
@@ -2,11 +2,19 @@ apiVersion: v1
+    - https://ibm.github.io/helm101/guestbook-0.1.0.tgz
     version: 0.1.0
-generated: 2018-05-10T21:50:41.83648523-04:00
+generated: 2019-02-21T14:09:27.909109099Z
```
Why we need to change anything in the 0.1.0 section?

hickeyma replied:
This is when `helm index` runs its updates the index file to add the new chart version (0.2.0) and changes the generated timestamp accordingly. I think this probably should changes as it has been regenerated the index file and is not associated to the v0.1.0 timestamp.

I have updated changed the "created" timestamp for chart v0.1.0 back to its original.

spzala commented on index.yaml:
```diff
@@ -2,11 +2,19 @@ apiVersion: v1
+    - https://ibm.github.io/helm101/guestbook-0.1.0.tgz
     version: 0.1.0
-generated: 2018-05-10T21:50:41.83648523-04:00
+generated: 2019-02-21T14:09:27.909109099Z
```
Why we need to change anything in the 0.1.0 section?

hickeyma replied:
Same as above in https://github.com/IBM/helm101/pull/22#discussion_r260414874

hickeyma committed:
Update after review

Update following review comments:
- https://github.com/IBM/helm101/pull/22#pullrequestreview-207634049

hickeyma commented:
@spzala Thanks for review. Comments updated and ready for review again.

spzala commented:
Thanks @hickeyma I haven't tested the new chart but the changes lgtm

hickeyma commented:
Thanks @spzala
